interfaces/cython/cantera/reactor.pyx

```diff
@@ -727,10 +727,10 @@ cdef class MassFlowController(FlowDevice):
 
 cdef class Valve(FlowDevice):
     r"""
-    In Cantera, a `Valve` is a flow devices with mass flow rate that is a
+    In Cantera, a `Valve` is a flow device with mass flow rate that is a
     function of the pressure drop across it. The default behavior is linear:
 
-    .. math:: \dot m = K_v (P_1 - P_2)
```

Comment:
Well, that's kind of a built-in ambiguity in normal mathematical notation, but I see your point in this case since we're trying to draw a distinction between the two. I think one reason I avoided `\cdot` previously is that it's uglier when you're looking at the plain text version, e.g. while reading the help in IPython/Jupyter, but doing something to disambiguate here makes sense. No idea why a capital `F` is used in the second formula either, as lowercase `f` is more likely to be read as "a function".